backend/src/data_2/config.json

```diff
@@ -0,0 +1 @@
+{"update_date": "2020-11-14 01:00:04"}
```

Comment:
seriously, let's remove config.json from repo (it is already mentioned in .gitignore, but it has not been manually removed from repository since)